src/pretalx/cfp/views/wizard.py

```diff
@@ -96,6 +98,12 @@ def done(self, request, draft=False, steps=None):
                 step.done(request, draft=draft)
 
         if not draft:
+            submission_state_change.send_robust(
+                request.event,
+                submission=request.submission,
+                old_state=SubmissionStates.DRAFT,
```

Comment:
On the one hand side i believe, this would have been OK: the submission is de facto in a draft state before it is submitted, even if the draft state was never stored in the database... Anyway, supporting `None` for the `old_state` is definitely more clean.

Reply:
Also absolutely right approach to keep that in the model.